### PR TITLE
[ISSUE] #14 Update translation.json

### LINKED ISSUE #14: 🤔 [ISSUE] - Fix Typo in NLP Trainer
### Description

image
We should put "Type" instead of "Lookup Strategy" to indicate the type of dataset the sample should be in (Train or Test).

### Reproduction steps

```bash
1. Login
2. Go to 'NLP' in the left sidebar
```

### Logs

_No response_

### Browsers

_No response_

### OS

_No response_

### PR DESCRIPTION
**Pull request by kartik1112**
# Motivation

Changed string values for "training" --> "train" and "Lookup Stratergies" --> "Type"

Fixes #14 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (Storybook)
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: frontend/src/i18n/en/translation.json [hunk 1]
```diff
@@ -324,7 +324,7 @@
     "value": "Value",
     "synonyms": "Synonyms",
     "lookups": "Lookups",
-    "lookup_strategies": "Lookup Strategy",
+    "lookup_strategies": "Type",
```
Comment (marrouchi) [inline frontend/src/i18n/en/translation.json:327]: Amazing ! 🥳 Thanks for your contribution.

lookup_strategies is used elsewhere (in the NLP entity Dialog form). I suggest you add a new string.

If you could also add the same in fr/translation.json for the french translation (Type goes also for french).

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: frontend/src/components/nlp/components/NlpTrainForm.tsx [hunk 2]
```diff
@@ -198,7 +198,7 @@ const NlpDatasetSample: FC<NlpDatasetSampleProps> = ({
               {t("title.nlp_train")}
             </Typography>
             <FormControl>
-              <FormLabel>{t("label.lookup_strategies")}</FormLabel>
+              <FormLabel>{t("label.lookup_type")}</FormLabel>
```
Comment (marrouchi) [inline frontend/src/components/nlp/components/NlpTrainForm.tsx:201]:
```suggestion
              <FormLabel>{t("label.type")}</FormLabel>
```

File: frontend/src/i18n/en/translation.json [hunk 3]
```diff
@@ -324,6 +324,7 @@
     "value": "Value",
     "synonyms": "Synonyms",
     "lookups": "Lookups",
+    "lookup_type": "Type",
```
Comment (marrouchi) [inline frontend/src/i18n/en/translation.json:327]:
```suggestion
    "type": "Type",
```

File: frontend/src/i18n/fr/translation.json [hunk 4]
```diff
@@ -323,6 +323,7 @@
     "nlp_entity_value": "Valeur TALN",
     "value": "Valeur",
     "lookups": "Stratégies",
+    "lookup_type": "Type",
```
Comment (marrouchi) [inline frontend/src/i18n/fr/translation.json:326]:
```suggestion
    "type": "Type",
```

**Review by marrouchi — CHANGES_REQUESTED** [review]

**Review by marrouchi — APPROVED** [review]

File: frontend/src/i18n/fr/translation.json [hunk 5]
```diff
@@ -543,7 +544,7 @@
     "total": "Totale",
     "general": "Général",
     "other": "Autre",
-    "no_data":" Pas de données"
+    "no_data": " Pas de données"
```
Comment (yassinedorbozgithub) [inline frontend/src/i18n/fr/translation.json:547]:
```suggestion
    "no_data": "Pas de données"
```
Hi @kartik1112, can you please remove the extra space in the beginning of this value.

**Review by yassinedorbozgithub — CHANGES_REQUESTED** [review]

File: frontend/src/i18n/en/translation.json [hunk 6]
```diff
@@ -544,7 +545,7 @@
     "total": "Total",
     "general": "General",
     "other": "Other",
-    "no_data":" No data"
+    "no_data": " No data"
```
Comment (yassinedorbozgithub) [inline frontend/src/i18n/en/translation.json:548]:
```suggestion
    "no_data": "No data"
```
Hi @kartik1112, can you please remove the extra space in the beginning of this value.

**Review by yassinedorbozgithub — CHANGES_REQUESTED** [review]

**Review by yassinedorbozgithub — APPROVED** [review]
LGTM :+1:

**Pull request merged**